rtv/content.py

```diff
@@ -131,9 +131,12 @@ def strip_praw_submission(sub):
         data['flair'] = flair
         data['url_full'] = sub.url
 
+        if flair is not None and flair[0] != '[':
```

Comment:
This will crash if the flair is an empty string, I would suggest using flair.startswith() instead.

Do some of the flairs already have brackets added? I haven't seen any.